### ISSUE TITLE
[BUG] A11y - Ensures every HTML document has a lang attribute

### ISSUE THREAD

**Issue by YuriDevAT** [YurisCodingClub/accessibility-mentor#35]
### Description
When running the a11y test 
```
npx playwright test
```

following error occurs
```
Object {
+     "description": "Ensures every HTML document has a lang attribute",
+     "help": "<html> element must have a lang attribute",
+     "helpUrl": "https://dequeuniversity.com/rules/axe/4.8/html-has-lang?application=playwright",
+     "id": "html-has-lang",
+     "impact": "serious",
+     "nodes": Array [
+       Object {
+         "all": Array [],
+         "any": Array [
+           Object {
+             "data": Object {
+               "messageKey": "noLang",
+             },
+             "id": "has-lang",
+             "impact": "serious",
+             "message": "The <html> element does not have a lang attribute",
+             "relatedNodes": Array [],
+           },
+         ],
+         "failureSummary": "Fix any of the following:
+   The <html> element does not have a lang attribute",
+         "html": "<html>",
+         "impact": "serious",
+         "none": Array [],
+         "target": Array [
+           "html",
+         ],
+       },
+     ],
+     "tags": Array [
+       "cat.language",
+       "wcag2a",
+       "wcag311",
+       "TTv5",
+       "TT11.a",
+       "EN-301-549",
+       "EN-9.3.1.1",
+       "ACT",
+     ],
+   },
```

### Acceptance requirement

- [ ] add `lang` attribute to dismiss the error

**Comment by YuriDevAT** [comment]
Please go ahead @Chizycodes 🚀
